### PR TITLE
Init nextjs

### PR DESCRIPTION
**Pull request by nzcodarnoc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# korerorero-front-end
\ No newline at end of file
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
```
Comment (JacOng17) [inline README.md:6]: I tried both `npm run dev` & `yarn dev` & they both came up with errors & the server didn't start, so:

Either need to run `npm install` (I did this then the errors were fixed)
(or potentially `yarn install`? I didn't try this tho).

**Review by JacOng17 — CHANGES_REQUESTED** [review]
Have left a comment that might need fixing on the README, else it's good to go :+1:

**Comment by JacOng17** [comment]
image

File: README.md (hunk 1 above)
Comment (nzcodarnoc) [reply README.md:6]: Good spotting. I've updated the readme.

**Review by JacOng17 — APPROVED** [review]
LGTM, README updated with instructions to run `yarn install` before `yarn dev` :+1:

**Pull request merged**